src/client/gamepad.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#ifdef GAMEPAD_SUPPORT
```

Comment:
In my opinion, instead of disabling the code itself, you could do the ifdef where the code is included an not supported

Reply:
Well, `include_directory` is used, so the `gamepad.cpp` file is disabled, yet the gampad header will always be included even if support is disabled (ex. no selective `include_directory()` that I know of)

Reply:
Header maybe will be in the include path, but in no way does that mean it's included. Yes, do not add this #ifdef here.

Reply:
No-- I've just checked this. `GAMEPAD_SUPPORT` is disabled, and as it is in the include path, `gamepad.cpp` is not set to compile, and you will still get:

```
/Users/keegan/Documents/mcpelauncher-linux/src/client/gamepad.h:7:10: fatal error:
      'libudev.h' file not found
#include <libudev.h>
```

Looking into this

Reply:
`main.cpp` includes it (facepalm)